palette/src/hues.rs

```diff
@@ -20,7 +20,7 @@ macro_rules! make_hues {
         /// number (like `f32`). This makes many calculations easier, but may
         /// also have some surprising effects if it's expected to act as a
         /// linear number.
-        #[derive(Clone, Copy, Debug, Default)]
+            #[derive(Clone, Copy, Debug, Default)]
```

Comment:
`cargo fmt` does weird indenting here.

Reply:
Indeed, what is this? Maybe some bug. You can try to shuffle the attributes around, but if this is what it want to do, then so be it. 🤷‍♂️

Reply:
I saw some other odd thing further down, so maybe it's not a big fan of macros at the moment.

Reply:
Playing around with it more, it seems to just be not opinionated inside the macro. If I manually correct the indentation, `rustfmt` doesn't "fix" it.

Reply:
I would expect it to not want to do too much with macro code, considering it's much more free form. 🤔 Just do your best to make it look nice.